<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
           <widget class="QLineEdit" name="EditPass">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="EditUser"/>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="pushButton_exit">
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_login">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="displayService">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="serviceOffers">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="images.qrc">:/resourcesImages/pamphlet cyberSec.jpeg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_basic">
          <property name="text">
           <string>Buy Basic Service - $19.99</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_business">
          <property name="text">
           <string>Buy Business Service - $49.99</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_enterprise">
          <property name="text">
           <string>Buy Enterprise Service - $79.99</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_backToLogin">
          <property name="text">
           <string>Back to Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainMenu">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="6" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_customers">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="viewKeyLabel">
            <property name="text">
             <string>2 - View Key Customers:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_addRemove">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="viewLabel">
            <property name="text">
             <string>1 - View Customers:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="addLabel">
            <property name="text">
             <string>3 - Add or Remove Customer:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_keyCustomers">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="exitLabel">
            <property name="text">
             <string>5 - Exit</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButton_exit_2">
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="menuLabel">
            <property name="text">
             <string>Main Menu</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>4 - Back to Login</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ViewCustomer_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_updateCustomer">
            <property name="text">
             <string>Update Customer</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_generateList">
            <property name="text">
             <string>Customer List</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Customer List:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_viewCustomer">
            <property name="text">
             <string>View Customer</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_back">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Customer Info:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="CustomerList"/>
          </item>
          <item row="1" column="1">
           <widget class="QListWidget" name="CustomerInfo"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="UpdateCustomer">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="rating"/>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Key Member:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_update">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Rating:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QListWidget" name="customerInfo"/>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_back_2">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QRadioButton" name="isKeyButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ViewKeyCustomer">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Key Customers:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QListWidget" name="KeyData"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_generateKey">
            <property name="text">
             <string>Show List</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_view">
            <property name="text">
             <string>View Customer</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_back_3">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="KeyList"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AddRemove">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="5" column="2">
           <widget class="QLineEdit" name="lineEdit_address_2"/>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Address Line 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="10">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="2">
           <widget class="QLineEdit" name="lineEdit_address_1"/>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_customerName"/>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Address Line 1</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_back_4">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="1" colspan="2">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Customer Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Enter Customer Information</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_remove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_add">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" rowspan="10">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="lineEdit_password"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="lineEdit_rating"/>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Rating</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Key Customer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuContact_us">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionContact_us"/>
   </widget>
   <addaction name="menuContact_us"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionContact_us">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resourcesImages/contact-phone-icon-4.png</normaloff>:/resourcesImages/contact-phone-icon-4.png</iconset>
   </property>
   <property name="text">
    <string>Contact us</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>EditUser</tabstop>
  <tabstop>EditPass</tabstop>
  <tabstop>pushButton_login</tabstop>
  <tabstop>pushButton_exit</tabstop>
  <tabstop>pushButton_customers</tabstop>
  <tabstop>pushButton_keyCustomers</tabstop>
  <tabstop>pushButton_addRemove</tabstop>
  <tabstop>pushButton_exit_2</tabstop>
  <tabstop>pushButton_generateList</tabstop>
  <tabstop>pushButton_viewCustomer</tabstop>
  <tabstop>pushButton_updateCustomer</tabstop>
  <tabstop>pushButton_back</tabstop>
  <tabstop>customerInfo</tabstop>
  <tabstop>rating</tabstop>
  <tabstop>pushButton_update</tabstop>
  <tabstop>pushButton_back_2</tabstop>
  <tabstop>KeyData</tabstop>
  <tabstop>pushButton_generateKey</tabstop>
  <tabstop>pushButton_view</tabstop>
  <tabstop>pushButton_back_3</tabstop>
  <tabstop>lineEdit_customerName</tabstop>
  <tabstop>lineEdit_address_1</tabstop>
  <tabstop>lineEdit_address_2</tabstop>
  <tabstop>pushButton_back_4</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
